addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/XsltInjectionScanRule.java

```diff
@@ -170,7 +201,6 @@ private Boolean tryInjection(String param, XSLTInjectionType checkType) {
                         getName(),
                         e.getClass().getName(),
                         e.getMessage());
-                continue;
```

Comment:
We no longer want it to go on with other payloads?

Reply:
This statement actually has no effect - it was flagged by SonarLint ;)

Reply:
Okay